UnitTests/LineCanvasTests.cs

```diff
@@ -218,6 +218,68 @@ public void TestLineCanvas_Window_Double ()
 			TestHelpers.AssertDriverContentsAre (looksLike, output);
 		}
 
+
+		[Theory, AutoInitShutdown]
+		[InlineData(BorderStyle.Single)]
+		[InlineData(BorderStyle.Rounded)]
+		public void TestLineCanvas_Window_DoubleTop_SingleSides (BorderStyle thinStyle)
+		{
+			var v = GetCanvas (out var canvas);
+
+			// outer box
+			canvas.AddLine (new Point (0, 0), 9, Orientation.Horizontal, BorderStyle.Double);
+			canvas.AddLine (new Point (9, 0), 4, Orientation.Vertical, thinStyle);
+			canvas.AddLine (new Point (9, 4), -9, Orientation.Horizontal, BorderStyle.Double);
+			canvas.AddLine (new Point (0, 4), -4, Orientation.Vertical, thinStyle);
+
+
+			canvas.AddLine (new Point (5, 0), 4, Orientation.Vertical,thinStyle);
+			canvas.AddLine (new Point (0, 2), 9, Orientation.Horizontal, BorderStyle.Double);
+
+			v.Redraw (v.Bounds);
+
+			string looksLike =
+@"    
+╒════╤═══╕
```

Comment:
I don't see "DoubleTop" here??

Reply:
Yea, seems not all orientations are working:

<img width="156" alt="image" src="https://user-images.githubusercontent.com/585482/218237435-348ee574-a04f-484a-9a7e-5b6949c4c02a.png">

Reply:
Hmn weird... Did you definitely `git pull` the latest? Should be a443ec7?

I did the corners first and only got to the Ts in a443ec7 so this behavior would be consistent with the older commit f716dbe596c7086754278aba54e36d3a63b199b4 only.

Reply:
![Uploading shot-2023-02-11_08-11-04.jpg…]()

Reply:
![shot-2023-02-11_08-11-04](https://user-images.githubusercontent.com/31306100/218247931-0f3aaa50-15c7-4611-8194-d6056244646a.jpg)

Reply:
I did

<img width="513" alt="image" src="https://user-images.githubusercontent.com/585482/218289080-86b0a3e9-6522-4386-8cc6-67adcd494161.png">

This is a direct link to the test that seems incorrect in this PR:

https://github.com/tznind/gui.cs/blob/d555576a138d222313c0efe8f35c2149eed196a6/UnitTests/Core/LineCanvasTests.cs#L243

This is what i see when look at your fork with this branch:

<img width="617" alt="image" src="https://user-images.githubusercontent.com/585482/218289129-d2d55162-86f0-4b65-a284-8b71cc394aea.png">